src/sardana/macroserver/macros/standard.py

```diff
@@ -648,17 +650,24 @@ class ct(Macro, Hookable):
     env = ('ActiveMntGrp',)
     hints = {'allowsHooks': ('pre-acq', 'post-acq')}
     param_def = [
-        ['integ_time', Type.Float, 1.0, 'Integration time']
+        ['integ_time', Type.Float, 1.0, 'Integration time'],
+        ['mnt_grp', Type.MeasurementGroup, Optional, 'MntGrp to use']
```

Comment:
I would not use accronyms in the description.

Reply:
Provided in [43eabb4](https://github.com/sardana-org/sardana/pull/940/commits/43eabb45148e4ea0512a146f7b7cb7577bf0b237) commit.